input/fsh/CGImplications.fsh

```diff
@@ -30,26 +30,35 @@ Description:    "Properties common to genomic implications expressed as computab
 // * component ^slicing.rules = #open
 // * component ^slicing.description = "Slice based on the component.code pattern"
 * component contains
-    evidence-level 0..*
+    evidence-level 0..* and
+	clinical-significance 0..1
```

Comment:
NIT: looks like formatting is off (line 34 doesn't align with line 33 and others).

Reply:
resolved